geospatial/dted.ksy

```diff
@@ -0,0 +1,626 @@
+        type: angle_ddmmssh
+        doc: |
+          Latitude of coordinate pair, leading zero for values less than 10; |
+          H is the hemisphere of the data. (DDMMSSH)
+      - id: longitude
+        type: angle_dddmmssh
+        doc: |
+          Longitude of coordinate pair, leading zeroes for values less than |
+          100; H is the hemisphere of the data (DDDMMSSH).
+  angle_ddmmss_sh:
+    doc: |
+      Typed used created to convert to degree decimal representation
+    seq:
+      - id: degrees
+        type: str
+        size: 2
+      - id: minutes
+        type: str
+        size: 2
+      - id: seconds
+        type: str
+        size: 2
+      - id: decimal_point
+        type: str
+        size: 1
+      - id: tenths_of_seconds
+        type: str
+        size: 1
+      - id: hemisphere
+        type: str
+        size: 1
+    instances:
+      value:
+        value: (degrees.to_i + minutes.to_i/60.0 + seconds.to_i/3600.0 + tenths_of_seconds.to_i/36000.0) * (hemisphere == 'N'? 1 : -1)
+        doc: |
+          Degree decimal floating point representation        
+  angle_dddmmss_sh:
+    doc: |
+      Typed used created to convert to degree decimal representation
+    seq:
+      - id: degrees
+        type: str
+        size: 3
+      - id: minutes
+        type: str
+        size: 2
+      - id: seconds
+        type: str
+        size: 2
+      - id: decimal_point
+        type: str
+        size: 1
+      - id: tenths_of_seconds
+        type: str
+        size: 1
+      - id: hemisphere
+        type: str
+        size: 1
+    instances:
+      value:
+        value: (degrees.to_i + minutes.to_i/60.0 + seconds.to_i/3600.0 + tenths_of_seconds.to_i/36000.0) * (hemisphere == 'N'? 1 : -1)
+        doc: |
+          Degree decimal floating point representation        
+  angle_ddmmssh:
+    doc: |
+      Typed used created to convert to degree decimal representation
+    seq:
+      - id: degrees
+        type: str
+        size: 2
+      - id: minutes
+        type: str
+        size: 2
+      - id: seconds
+        type: str
+        size: 2
+      - id: hemisphere
+        type: str
+        size: 1
+    instances:
+      value:
+        value: (degrees.to_i + minutes.to_i/60.0 + seconds.to_i/3600.0) * (hemisphere == 'N'? 1 : -1)
+        doc: |
+          Degree decimal floating point representation        
+  angle_dddmmssh:
+    doc: |
+      Typed used created to convert to degree decimal representation
+    seq:
+      - id: degrees
+        type: str
+        size: 3
+      - id: minutes
+        type: str
+        size: 2
+      - id: seconds
+        type: str
+        size: 2
+      - id: hemisphere
+        type: str
+        size: 1
+    instances:
+      value:
+        value: (degrees.to_i + minutes.to_i/60.0 + seconds.to_i/3600.0) * (hemisphere == 'N'? 1 : -1)
+        doc: |
+          Degree decimal floating point representation
+  date_yymm:
+    seq:
+      - id: year
+        type: str
```

Comment:
Is it an int?

Reply:
It is an int encoded as ASCII.  As such I'll change it as follows:
```
  date:
    seq:
      - id: year_text
        type: str
        size: 2
      - id: month_text
        type: str
        size: 2
    instances:
      year:
        value: year_text.to_i
      month:
        value: month_text.to_i
```

Reply:
Why not to use the existing type, just parameterizing it with string length?